package.json

```diff
@@ -901,7 +901,7 @@
     "react-final-form-listeners": "^1.0.2",
     "react-router-prop-types": "^1.0.4",
     "redux-form": "^8.3.7",
-    "zustand": "^3.7.2"
+    "zustand": "^4.1.1"
```

Comment:
If it's provided by platform-complete, can we move it to `peerDependencies`?

Reply:
@mariia-aloshyna and @BogdanDenis if this was moved to peer please also remove it from the `dependecies` list and add it to `devDependencies`.

Reply:
Hi @mkuklis done, thank you!